constants/providersContent.ts

```diff
@@ -50,84 +52,171 @@ const REAL_HARDWARE: ProvidersSection = {
   id: realHardware,
   title: 'Run With Real Hardware',
   description: 'Qiskit allows for easy research and development for specific industry use cases that have the highest potential for quantum advantage.',
-  // linkset: [
-  //   {
-  //     label: 'See machine learning docs',
-  //     url: 'https://qiskit.org/documentation/machine-learning/'
-  //   },
-  //   {
-  //     label: 'See nature docs',
-  //     url: 'https://qiskit.org/documentation/nature/'
-  //   },
-  //   {
-  //     label: 'See finance docs',
-  //     url: 'https://qiskit.org/documentation/finance/'
-  //   },
-  //   {
-  //     label: 'See optimization docs',
-  //     url: 'https://qiskit.org/documentation/optimization/'
-  //   }
-  // ],
   subSections: [
     {
-      title: 'IBM Quantum Qiskit Runtime',
-      content: {
-        image: '/images/providers/applications/optimization.png',
-        description: 'The Qiskit Optimization package covers the whole range from high-level modeling of optimization problems, with automatic conversion of problems to different required representations, to a suite of easy-to-use quantum optimization algorithms that are ready to run on classical simulators, as well as on real quantum systems.',
-        cta: {
-          label: 'Solve the Max Cut Problem',
-          url: 'https://qiskit.org/documentation/optimization/tutorials/01_quadratic_program.html'
-        }
-      }
+      title: 'IBM Qiskit Runtime',
```

Comment:
I am OK with putting content directly to `.ts` But I also want to let you know that the long term plan is to be able to populate these fields using the Ecosystem infrastructure. People should be able to submit their own providers as an Qiskit Ecosystem project with extra requirement specific to providers (e.g. hello world examples). With that in mind, would it be better to write the content in `.ts` or in json similar to the ecosystem?

Reply:
Oh, this is good information to know... 

@y4izus what do you think? 

Seems like, at some point (whether no or in another PR), we should refactor the code so we can accept JSON... especially if it will be similar to how the Ecosystems pages are populated.

Thanks @HuangJunye

Reply:
Per offline conversations we decided to do the JSON file now